setup.py

```diff
@@ -392,7 +392,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
 leveldb = ['plyvel']
 mongo = [
     'dnspython>=1.13.0,<3.0.0',
-    'pymongo>=3.6.0',
```

Comment:
1. We should add the comment with the reason of why we are limiting mongo
2. Can we fix it instead maybe?

Reply:
1. Added comment
2. There are quite a number of breaking changes in pymongo `4.0.0`. I think we should limit the major version first. We may migrate to pymongo `4.0.0` in the future PR.
https://pymongo.readthedocs.io/en/stable/changelog.html#breaking-changes-in-4-0

Reply:
Oh yeah. That list looks impressive.